server/views/partials/head.njk

```diff
@@ -35,13 +35,12 @@
 
 {% if googleAnalyticsId %}
   <!-- Google Tag Manager -->
-  <script nonce="{{ cspNonce }}">window.dataLayer = window.dataLayer || [];</script>
-
-  <script nonce="{{ cspNonce }}">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-    'https://www.googletagmanager.com/gtm.js?id='+i+dl;var n=d.querySelector('[nonce]');
-    n&&j.setAttribute('nonce',n.nonce||n.getAttribute('nonce'));f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','{{ googleAnalyticsId }}');</script>
-  <!-- End Google Tag Manager -->
+  <script nonce="{{ cspNonce }}" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId }}"></script>
```

Comment:
copied from the installation instructions and was actually the code that was in use before copying from the typescript template - think I'll need to do a backport of the code...